test/integration/minishift.go

```diff
@@ -190,6 +190,43 @@ func (m *Minishift) executingMinishiftCommand(command string) error {
 	return nil
 }
 
+func (m *Minishift) setImageCaching(operation string) error {
```

Comment:
can we please modify this to:
```go
func (m *Minishift) setImageCaching(operation string) error {
    return m.executingMinishiftCommand(fmt.Sprintf("config set image-caching %s", operation))
}
```
writing `setImageCaching("true")` kinda feels more intuitive to me.
I haven't gone through the feature files.

Reply:
This is fine, as it is aligned with the step `And image caching is disabled`.